src/pages/index.tsx

```diff
@@ -1,17 +1,158 @@
-import { Inter } from "next/font/google";
-
-const inter = Inter({ subsets: ["latin"] });
-import { LogoNerdbord } from "../components/LogoNerdbord";
-
+import { MenuItem } from "@/components/MenuItem";
+import { SidebarMenu } from "@/components/SidebarMenu";
+import React, { MutableRefObject, useEffect, useRef } from "react";
 import styles from "../styles/index.module.css";
 
 export default function Home() {
+  const ref1: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const ref2: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const ref3: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const ref4: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const ref5: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const ref6: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const ref7: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+
+  const navLink1: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const navLink2: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const navLink3: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const navLink4: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const navLink5: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const navLink6: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+  const navLink7: MutableRefObject<HTMLElement | undefined> = useRef(undefined);
+
+  const imgs = [ref1, ref2, ref3, ref4, ref5, ref6, ref7];
+  const navLinks = [
+    navLink1,
+    navLink2,
+    navLink3,
+    navLink4,
+    navLink5,
+    navLink6,
+    navLink7,
+  ];
+
+  useEffect(() => {
+    window.onscroll = () => {
```

Comment:
**issue:** You are not removing `onscroll` handler when unmounting component - it can cause memory leaks